v2/parser/apis/api/apienc/errors.go

```diff
@@ -86,6 +86,6 @@ var (
 
 	errHTTPStatusFieldMustBeInt = errRange.New(
 		"Invalid response type",
-		"Fields tagged with encore:\"httpstatus\" must be of type int.",
+		"Fields tagged with encore:\"httpstatus\" must be of an integer type (int, int16, int32, int64, uint, uint16, uint32, uint64).",
```

Comment:
I think it's fine to just say:
```suggestion
		"Fields tagged with encore:\"httpstatus\" must be of an integer type."
```